pagopa/io-app
# chore: [IA-277] Remove services from persisted store configuration

## Short description
This PR aims to remove services from entities' `redux-persist` configuration.

## List of changes proposed in this pull request
- Add `services` to entities' configuration blacklist
- Add a migration rule to remove old stored services

## How to test
Run app and check if the migration rule has been executed and check the store won't persist services anymore.
